project/profiles/capacity4more/behat/features/group_dashboard.feature

```diff
@@ -135,3 +135,10 @@ Feature: Group dashboard
     | galileo     |
     | badhairday  |
     | president   |
+
+  @api
+  Scenario: Highlighted document content title should contain a link.
+    Given I am logged in as user "badhairday"
+    When  I visit the dashboard of group "Nobel Prize"
+    Then  I should see the text "Nobel Prize"
+    And I should see the link "Nobel Prize sheet"
```

Comment:
```gherkin
When I visit the highlighted link "Nobel Prize sheet"
Then I should get 200 response
```

Visit highlighted link should visit a link within the highlight box class only whatever it is.